cmd2/constants.py

```diff
@@ -12,6 +12,7 @@
 REDIRECTION_APPEND = '>>'
 REDIRECTION_CHARS = [REDIRECTION_PIPE, REDIRECTION_OUTPUT]
 REDIRECTION_TOKENS = [REDIRECTION_PIPE, REDIRECTION_OUTPUT, REDIRECTION_APPEND]
+COMMENT_CHAR = '#'
```

Comment:
There are relatively few characters in the ASCII set which could serve as a comment character.  If we could comment here something about possibilities, it might be helpful.

Reply:
Since this character has to appear first in a command line, I'm OK with not suggesting the user ever change it. In reality I like # because it has connections to Python and Bash. Plus our docs explicitly reference #.